src/Snackbar.js

```diff
@@ -0,0 +1,75 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+// This component doesn't use the javascript from MDL.
+// This is the expected behavior and the reason is because it's not written in
+// a way to make it easy to use with React.
+const ANIMATION_LENGTH = 250;
+
+class Snackbar extends React.Component {
+    static propTypes = {
+        action: PropTypes.string,
+        active: PropTypes.bool.isRequired,
+        className: PropTypes.string,
+        onClick: PropTypes.func,
```

Comment:
Personally, when I saw onClick, I thought it was a handler when snackbar is clicked (not just the action button). I think we should keep mdl naming (actionHandler). Or if we want to stick to `on-` naming, maybe onActionClick?